package proto_ff;

import "yd_fieldoptions.proto";


message E_AssistAssistbox
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 m_times = 2[(yd_fieldoptions.field_cname) = "需要协助次数"];
	optional int32 m_itemid = 3[(yd_fieldoptions.field_cname) = "礼包物品id"];
}


message Sheet_AssistAssistbox
{
	repeated E_AssistAssistbox E_AssistAssistbox_List = 1[(yd_fieldoptions.field_arysize)=20];
}

message E_AssistAssistthank
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional string m_msg = 2[(yd_fieldoptions.field_cname) = "消息", (yd_fieldoptions.field_bufsize) = 60];
}


message Sheet_AssistAssistthank
{
	repeated E_AssistAssistthank E_AssistAssistthank_List = 1[(yd_fieldoptions.field_arysize)=20];
}
